redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -169,7 +177,6 @@ internal open class UIViewLazyList(
     tableView.apply {
       dataSource = this@UIViewLazyList.dataSource
       delegate = tableViewDelegate
-      prefetchingEnabled = true
```

Comment:
This doesn’t work on iOS 14

Reply:
Also defaults to YES anyway!
https://developer.apple.com/documentation/uikit/uitableview/3801922-prefetchingenabled